connector/connect/server/pom.xml

```diff
@@ -188,6 +188,17 @@
       <artifactId>grpc-protobuf</artifactId>
       <version>${io.grpc.version}</version>
     </dependency>
+    <dependency>
```

Comment:
"`io.grpc.protobuf.lite.ProtoLiteUtils` has been moved to the `grpc-protobuf-lite` module, but `protobuf-javalite` and `protobuf-java` cannot be used simultaneously because there would be a conflict with `com.google.protobuf.Any`."